projects/dotcms-js/src/lib/core/site.service.ts

```diff
@@ -178,11 +177,10 @@ export class SiteService {
     }
 
     private setCurrentSite(site: Site): void {
+        this.selectedSite = site;
         if (this.selectedSite) {
```

Comment:
is the `IF` here makes sense, now?

you're assigning it on the previous line

Reply:
true, I did this  change because we where updating the `_switchSite` while the `selectedSite ` was not updated yet.